jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/SimpleValueReader.java

```diff
@@ -1,20 +1,23 @@
 package com.fasterxml.jackson.jr.ob.impl;
 
-import static com.fasterxml.jackson.jr.ob.impl.ValueWriterLocator.*;
+import com.fasterxml.jackson.core.JsonParser;
```

Comment:
As per https://github.com/FasterXML/jackson/blob/master/contribution/jackson-coding-style-guide.md

(written fairly recently, to explain logic of things like import order)

import order needs to start with most general (JDK) and then to more specific (3rd party deps, core Jackson deps, finally project itself).

So let's try not to reorder these (I know IDEs like reordering things when making changes).

Reply:
np. Sorry, I use Eclipse for OSS work. But if someone else has formatter I'd be happy to add to (ask others to add) formatter on that page.

@JooHyukKim do you happen to have a formatter to share?